tap_hubspot/hubspot.py

```diff
@@ -14,6 +14,50 @@ class RetryAfterReauth(Exception):
 
 LOGGER = singer.get_logger()
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+MANDATORY_PROPERTIES = {
```

Comment:
Have you confirmed with @JingLin0 or @iker-camara whether this includes all of the properties we need?